### ISSUE TITLE
API Upload Problems[BUG]

### ISSUE THREAD

**Issue by 1computerguy** [mitre/heimdall2#927]
I followed all the steps from the README to automate file uploads, but nothing works. I keep getting an error message:

`{"statusCode":401, "message":"Unauthorized"}`

Curl commands:
 - Generate access token:
`curl -X POST -H "Content-Type: application/json" -d '{"email": "user@example.com", "password": "password" }' http://localhost:3000/authn/login`

 - Attempt to upload
`curl -X POST -H "Content-Type: application/json" -H 'Authorization: Bearer  {"userID":"2", "accessToken":"<long access token string>"}' -d "@/home/vcscan/scans/02102021/<filename>.json" http://localhost:3000/evaluations`

I tried different variations of the upload command, omitting the "Authorization" or "Bearer" words, just using the token json, etc. and it either provides no output (with no upload), or it gives the error message above.

**Comment by rbclark** [comment]
The access token is only the `accessToken` section of the returned JSON. You can extract is programmatically into a bash variable by running:

`HEIMDALL_BEARER=$(curl -s -X POST -H "Content-Type: application/json" -d '{"email": "user@example.com", "password": "password" }' http://localhost:3000/authn/login | jq -r '.accessToken')`

This can then be used by saying 
`curl -X POST -H "Content-Type: application/json" -H "Authorization: Bearer $HEIMDALL_BEARER" -d "@filepath.json" http://localhost:3000/evaluations`

However there have been some changes to the evaluations API recently to add more fields which is making it much more difficult to upload via curl. This will be addressed in future releases.

For now, the fields required to interface with the API are

```json
{
      "data": <the JSON object of the evaluation you wish to upload>,
      "filename": "<the filename which will show on Heimdall>",
      "userId": "<your userID from the authentication curl request>",
      "evaluationTags": []
}
```

Using axios this payload can be sent using:

```
axios.post('/evaluations', jsonPayload);
```

**Comment by camdenmoors** [comment]
Uploads are now using mutli-part forms to allow uploads with curl to fix this, for example:
```
curl -X POST -H "Content-Type: application/json" \
	-H "Authorization: Bearer <Your Bearer>" \
	-d "@/home/vcscan/scans/02102021/<filename>.json" \
	"http://localhost:8000/evaluations"
```

**Comment by rbclark** [comment]
This will be available as part of `v2.3.10` which will be released sometime next week.
